_projects/git_task_list.md

```diff
@@ -0,0 +1,127 @@
+---
+collaborating_projects:
+  - gsoc-prep-tasks
+desc: "List open tasks from various issue tracker. Can be filtered to only show
+  newcomer friendly tasks."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - https://github.com/coala/projects/issues/480
+  - https://github.com/coala/coala/issues/3615
+  - https://github.com/coala/coala/issues/4080
```

Comment:
Combining from multiple hosters would be an excellent feature, and would mean this project does improve coala workflows, because lots of newcomers dont find our gitlab projects (and would not look at gitlab if they can avoid it, because they are lazy and do not want to learn new things)